src/HieDb/Run.hs

```diff
@@ -106,14 +106,14 @@ optParser defdb
     = Options
   <$> strOption (long "database" <> short 'D' <> metavar "DATABASE"
               <> value defdb <> showDefault <> help "References Database")
-  <*> switch (long "trace" <> short 'v')
-  <*> switch (long "quiet" <> short 'q')
-  <*> switch (long "virtual-file" <> short 'f')
+  <*> switch (long "trace" <> short 'v' <> help "Print SQL queries being executed")
+  <*> switch (long "quiet" <> short 'q' <> help "Don't print progress messages")
+  <*> switch (long "virtual-file" <> short 'f' <> internal)
```

Comment:
This is not used anywhere at the moment, so hide it from the help message to avoid confusion.